webapp/src/features/event/EventRoot.tsx

```diff
@@ -61,7 +62,6 @@ const EventRoot = () => {
     const onCancelChanges = () => {
         setHasChanges(false);
         if (!event) return;
-        setEventScratchpad(event);
```

Comment:
Just outta curiosity, was this line breaking something before? Or just kinda not doing anything?

Reply:
I don't remember at all, will check tomorrow, please help deer brain

Reply:
That's definitely not supposed to be there.  Good catch, tysm